Source/WebKit/Shared/API/Cocoa/_WKHitTestResult.h

```diff
@@ -47,6 +47,7 @@ WK_CLASS_AVAILABLE(macos(10.12), ios(16.0))
 @property (nonatomic, readonly, copy) NSURL *absoluteLinkURL;
 @property (nonatomic, readonly) BOOL hasLocalDataForLinkURL WK_API_AVAILABLE(macos(15.0), ios(18.0), visionos(2.0));
 @property (nonatomic, readonly, copy) NSString *linkLocalDataMIMEType WK_API_AVAILABLE(macos(15.0), ios(18.0), visionos(2.0));
```

Comment:
I think we might not need to do this PR at all, actually.  The nullness of linkLocalDataMIMEType already exposes whether local data exists, and the suggested filename can be derived from absoluteImageURL by putting it into an NSURLRequest and accessing its suggestedFilename in the UI process like we do in ResourceResponse::platformSuggestedFilename.

Reply:
Actually, that might lose some information from the response's Content-Disposition or Content-Type header fields.  Let's keep this approach.

Reply:
Actually, let's expose a nullable NSURLResponse.  That'll have all the info needed for all the LocalData interfaces: hasLocalDataForLinkURL, linkLocalDataMIMEType, and the new one.

Reply:
done